docs/src/man/comparisons.md

```diff
@@ -14,11 +14,11 @@ The following table compares the main functions of DataFrames.jl with the Python
 
 ```python
 df = pd.DataFrame(
-        {'grp': [1,2,1,2,1,2], 'x': range(6,0,-1), 'y': range(4,10), 'z': range(3,9)},
+        {'grp': [1,2,1,2,1,2], 'x': range(6,0,-1), 'y': range(4,10), 'z': [3,4,5,6,7,None]},
```

Comment:
is it `None`or `NaN` in pandas (I am not an expert here)

Reply:
It converts to NaN on the way in: 
```python
>>> df = pd.DataFrame(
...         {'grp': [1,2,1,2,1,2], 'x': range(6,0,-1), 'y': range(4,10), 'z': [3,4,5,6,7,None]},
...         index = list('abcdef'))
>>> df
   grp  x  y    z
a    1  6  4  3.0
b    2  5  5  4.0
c    1  4  6  5.0
d    2  3  7  6.0
e    1  2  8  7.0
f    2  1  9  NaN
```

Alternatively, we could use `np.nan` instead if numpy is imported: 
```python
df = pd.DataFrame(
        {'grp': [1,2,1,2,1,2], 'x': range(6,0,-1), 'y': range(4,10), 'z': [3,4,5,6,7,np.nan]},
        index = list('abcdef'))
```

Maybe we should add the imports:
```python
import pandas as pd
import numpy as np
```

Any preference?

Reply:
Then I think it is ok to stay with `None`.

Reply:
OK, I have just put back `None` (was accidentally committed earlier). I have also re-indented the pandas code.